core/src/main/resources/reference.conf

```diff
@@ -29,4 +29,9 @@ health-monitor-dispatcher {
 
 gcs {
   bucketLogsMaxAge = "180"
+}
+
+submissionmonitor {
+  submissionPollInterval = 1m
```

Comment:
Same as the [workflow poll interval](https://github.com/broadinstitute/firecloud-develop/blob/dev/configs/rawls/rawls.conf.ctmpl#L53).